File: Problem Solving/CodeForces/ICPC Assiut University Training Sheet/Newcomers/C/C-Sheet #2-Loop/G.c
#include <stdio.h>

int main() {
  int t;
  scanf("%d", &t);
  long long fac =1 ;
  while(t--){
      int num;
      scanf("%d", &num);
      for(int i = 1 ; i <= num ; i++){
          fac *= i;
      }
      printf("%lld\n", fac);
      fac = 1;
  }
return 0;
}

File: Problem Solving/CodeForces/ICPC Assiut University Training Sheet/Newcomers/Java/C-Sheet #2-Loop/G.java
import java.util.Scanner;

public class G {
  public static void main(String[] args){
  Scanner input = new Scanner(System.in);
  int t = input.nextInt();
  long fac = 1;
  for(int i=1;i<=t;i++){
    int num = input.nextInt();
    for (int j=1;j<=num;j++){
      fac *= j;
    }
    System.out.println(fac);
    fac = 1;
  }
  input.close();
  }
}